tests/telemetry/test_telemetry.py

```diff
@@ -154,3 +156,78 @@ def test_logs_after_fork(run_python_code_in_subprocess):
 
     assert status == 0, err
     assert err == b""
+
+
+def test_app_started_error(test_agent_session, run_python_code_in_subprocess):
+    code = """
+import logging
+logging.basicConfig()
+
+from ddtrace import tracer
+from ddtrace.filters import TraceFilter
+
+class FailingFilture(TraceFilter):
+    def process_trace(self, trace):
+       raise Exception("Exception raised in trace filter")
+
+tracer.configure(
+    settings={
+        "FILTERS": [FailingFilture()],
+    }
+)
+
+# generate and encode span
+tracer.trace("hello").finish()
+"""
+    _, stderr, status, _ = run_python_code_in_subprocess(code)
+    assert status == 0, stderr
+    assert b"Exception raised in trace filter" in stderr
+
+    events = test_agent_session.get_events()
+
+    assert len(events) == 2
+
+    # Same runtime id is used
+    assert events[0]["runtime_id"] == events[1]["runtime_id"]
+    assert events[0]["request_type"] == "app-closing"
+    assert events[1]["request_type"] == "app-started"
+    assert events[1]["payload"]["error"]["code"] == 1
+    assert events[1]["payload"]["error"]["message"] == "error applying processor FailingFilture()"
+
+
+def test_integration_error(test_agent_session, run_python_code_in_subprocess):
+    code = """
+import sqlite3
+# patch() of the sqlite integration assumes this attribute is there
+# removing it should cause patching to fail.
+del sqlite3.connect
+
+from ddtrace import patch, tracer
+patch(raise_errors=False, sqlite3=True)
+tracer.trace("test").finish()
+tracer.flush()
+"""
+
+    stdout, stderr, status, _ = run_python_code_in_subprocess(code)
+
+    assert status == 0, stderr
+    if sys.version_info[0] < 3:
+        # not quite sure why python 2.7 has a different error here, probably a difference with how it loads modules
```

Comment:
This is because we're attempting to log errors without calling `logging.basicConfig()`. I'll fix this